rasa_core/train.py

```diff
@@ -212,47 +180,50 @@ def train_comparison_models(stories,
 
     for r in range(runs):
         logging.info("Starting run {}/{}".format(r + 1, runs))
+
         for i in exclusion_percentages:
             current_round = exclusion_percentages.index(i) + 1
+
             for policy_config in policy_configs:
                 policies = config.load(policy_config)
+
                 if len(policies) > 1:
                     raise ValueError("You can only specify one policy per "
                                      "model for comparison")
+
                 policy_name = type(policies[0]).__name__
                 output = os.path.join(output_path, 'run_' + str(r + 1),
                                       policy_name +
                                       str(current_round))
 
                 logging.info("Starting to train {} round {}/{}"
-                             " with {}% exclusion".format(
-                                                policy_name,
-                                                current_round,
-                                                len(exclusion_percentages),
-                                                i))
+                             " with {}% exclusion"
+                             "".format(policy_name, current_round,
+                                       len(exclusion_percentages), i))
 
                 train_dialogue_model(
-                            domain, stories, output,
-                            policy_config=policy_config,
-                            exclusion_percentage=i,
-                            kwargs=kwargs,
-                            dump_stories=dump_stories)
-
+                        domain, stories, output,
+                        policy_config=policy_config,
+                        exclusion_percentage=i,
+                        kwargs=kwargs,
+                        dump_stories=dump_stories)
 
-def get_no_of_stories(stories, domain):
 
+def get_no_of_stories(story_file, domain):
     """Get number of stories in a file."""
 
-    no_stories = len(StoryFileReader.read_from_folder(stories,
-                                                      TemplateDomain.load(
-                                                            domain)))
-    return no_stories
+    stories = StoryFileReader.read_from_folder(story_file,
+                                               TemplateDomain.load(domain))
+    return len(stories)
 
 
 def do_default_training(cmdline_args, stories, additional_arguments):
+    """Train a model."""
+
     if not cmdline_args.out:
         raise ValueError("you must provide a path where the model "
                          "will be saved using -o / --out")
+
     if (isinstance(cmdline_args.config, list) and
             len(cmdline_args.config) > 1):
         raise ValueError("You can only pass one config file at a time")
```

Comment:
We could do away with these checks by leaving `nargs` for `config` unspecified, in which case it is passed as a single string, not a list. The script won't even run if you'd specify more

Reply:
yes, that is a good point, at the moment we really do not need multiple configs for this script